src/components/app/pageLocalPolicy/us/statePage/recentActivitySection.tsx

```diff
@@ -0,0 +1,57 @@
+import { RecentActivityAndLeaderboardTabs } from '@/components/app/pageHome/us/recentActivityAndLeaderboardTabs'
+import { RecentActivity } from '@/components/app/pageLocalPolicy/common/statePage/recentActivitySection'
+import { Section } from '@/components/app/pageLocalPolicy/common/statePage/section'
+import { getPublicRecentActivity } from '@/data/recentActivity/getPublicRecentActivity'
+import { DEFAULT_SUPPORTED_COUNTRY_CODE } from '@/utils/shared/supportedCountries'
+import { getIntlUrls } from '@/utils/shared/urls'
+
+const RECENT_ACTIVITY_TITLE = 'Recent activity'
+const RECENT_ACTIVITY_BUTTON_LABEL = 'View all'
+
+const countryCode = DEFAULT_SUPPORTED_COUNTRY_CODE
+
+const urls = getIntlUrls(countryCode)
+
+const ITEMS_PER_PAGE = 5
+
+interface UsRecentActivitySectionProps {
+  stateCode: string
+  stateName: string
+}
+
+export async function UsRecentActivitySection({
+  stateCode,
+  stateName,
+}: UsRecentActivitySectionProps) {
+  const { count, data: publicRecentActivity } = await getPublicRecentActivity({
+    countryCode,
+    limit: ITEMS_PER_PAGE,
+    offset: 0,
+    stateCode,
+  })
+
+  return (
+    <Section>
+      <Section.Title>{RECENT_ACTIVITY_TITLE}</Section.Title>
+      <Section.SubTitle>{`See what actions people in ${stateName} are taking`}</Section.SubTitle>
```

Comment:
You don't need to interpolate a string for this, without it it's fairly more readable
```suggestion
      <Section.SubTitle>See what actions people in {stateName} are taking</Section.SubTitle>
```